doc/2-interface/menu-bar.md

```diff
@@ -54,25 +48,29 @@ items in _italic_ don't appear in basic mode and are only available in advanced
 
 - **exit**: closes Furnace.
 
-### export audio
+### export...
```

Comment:
I think we may have to move this to its own section...

Reply:
Where?

Reply:
How about I make it a page in `8-advanced`?

Reply:
Now doing so but on 2-interface.